### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by k-sharpe**

### CODE REVIEW

File: postal-injection/src/main/java/com/google/sps/servlets/StoreCSVInDataStore.java [hunk 1]
```diff
@@ -74,12 +74,17 @@ private void readCSVFile(File source, DatastoreService datastore) throws FileNot
...
-      String office = lineArray[17];
-      String warehouseLatitude = lineArray[18];
-      String warehouseLongitude = lineArray[19];
-      String distance = lineArray[20];
+      String city = lineArray[15];
+      String state = lineArray[16];
+      String customerLatitude = lineArray[17];
+      String customerLongitude = lineArray[18];
+      String timezone = lineArray[19];
+      String daylightSavingsFlag = lineArray[20];
```
Comment (jgao23) [inline postal-injection/src/main/java/com/google/sps/servlets/StoreCSVInDataStore.java:82]: why do you want this data?

File: postal-injection/src/main/java/com/google/sps/servlets/StoreCSVInDataStore.java [hunk 2]
```diff
@@ -74,12 +74,17 @@ private void readCSVFile(File source, DatastoreService datastore) throws FileNot
...
+      String geopointA = lineArray[21];
+      String geopointB = lineArray[22];
```
Comment (jgao23) [inline postal-injection/src/main/java/com/google/sps/servlets/StoreCSVInDataStore.java:84]: what's a geopoint

File: postal-injection/src/main/java/com/google/sps/servlets/StoreCSVInDataStore.java (hunk 1 above)
Comment (k-sharpe) [reply postal-injection/src/main/java/com/google/sps/servlets/StoreCSVInDataStore.java:82]: We don't need it but it was in the zipcode data csv so I kept it (more operation to remove)

File: postal-injection/src/main/java/com/google/sps/servlets/StoreCSVInDataStore.java (hunk 2 above)
Comment (k-sharpe) [reply postal-injection/src/main/java/com/google/sps/servlets/StoreCSVInDataStore.java:84]: Pretty much latitude and longitude, came with the new csv data

**Pull request merged**